README.md

```diff
@@ -1 +1,24 @@
-# iCare
+# icare
+
+## Project setup
+```
+npm install
+```
+
+### Compiles and hot-reloads for development
+```
+npm run serve
+```
+
+### Compiles and minifies for production
+```
+npm run build
+```
+
+### Lints and fixes files
+```
+npm run lint
+```
```

Comment:
I just want to point out that I suppose this command is to run ESlint to correct the file (if I am wrong please correct me). Actually we don't have to do that because ESlint would automatically running in the background when you are coding if you installed the extensions in VScode. Once there isn't any complain from ESlint, it means you are good to go.

Reply:
Yeah, I still left this in though in case someone is not using VScode and therefore doesn't have the extension.